### PR TITLE
removing facility now available!

### PR DESCRIPTION
**Pull request by DanoFecko**

### CODE REVIEW

File: gui/src/app/facilities/facilities-detail/facilities-detail.component.html [hunk 1]
```diff
@@ -44,20 +44,20 @@ <h4 class="mt-4">{{'FACILITIES.FACILITY_ATTRIBUTES' | translate}}</h4>
...
-      </button>
-      <button class="ml-2" mat-raised-button color="accent" (click)="addFacilityAdmin()">
-        {{'FACILITIES.ADD_ADMIN' | translate}}
-      </button>
-      <button class="ml-2" mat-raised-button color="accent" (click)="editAdmins()">
-        {{'FACILITIES.EDIT_ADMIN' | translate}}
-      </button>
-
+      <div class="mt-4">
+        <button class="ml-2" mat-raised-button color="accent" (click)="deleteFacility()">
```
Comment (Vojtech-Sassmann) [inline gui/src/app/facilities/facilities-detail/facilities-detail.component.html]: You should open a modal to let the user confirm if he truly wants to remove the facility. Also, the color of this button should be 'warning'.

File: gui/src/app/facilities/facilities-detail/facilities-detail.component.html [hunk 2]
```diff
@@ -44,20 +44,20 @@ <h4 class="mt-4">{{'FACILITIES.FACILITY_ATTRIBUTES' | translate}}</h4>
...
+          {{'FACILITIES.DELETE_FACILITY' | translate}}
+        </button>
+        <button class="ml-2" mat-raised-button color="accent" (click)="editFacility()">
```
Comment (Vojtech-Sassmann) [inline gui/src/app/facilities/facilities-detail/facilities-detail.component.html]: Please, change the colors of these buttons. Accent should be used only for those buttons that are probable to be used by a user. Make those buttons white and also, move them to the right side of the page. Take a look at the request-detail page.

File: gui/src/app/facilities/facilities-detail/facilities-detail.component.html (hunk 1 above)
Comment (Vojtech-Sassmann) [reply gui/src/app/facilities/facilities-detail/facilities-detail.component.html]: Also, the modal should not contain only 'are you sure?' but it should describe exactly, what the user is about to do. E.g. 'Are you sure you want to remove facility 'XYZ'?

**Comment by DanoFecko** [comment]
@Vojtech-Sassmann I have made requested changes

**Pull request merged**